amplify/backend/function/BorrowFromSchedule/src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "borrowfromschedule",
+  "version": "2.0.0",
+  "description": "Lambda function generated by Amplify",
+  "main": "./ts-output/handlers/api/CreateBatch.handler",
```

Comment:
Think it should be `./ts-output/handers/api/BorrowFromSchedule.js`

This is not part of this MR, but can you check the other packages, to make sure they point to the correct file as well?